RGTichy/CD Ch1 p16 workbook/CD Ch1 final.js

```diff
@@ -0,0 +1,10 @@
+
+console.log(fn(3,5,17,9))
```

Comment:
Instead of using function names like "fn" or "fnx", try using things like the title of the algorithm. We want to be writing code that anyone can look at and know pretty immediately what you're trying to achieve. Same thing with variable names other than "i" when you're iterating through something.